src/Simplex/Chat/Types.hs

```diff
@@ -351,15 +351,16 @@ pattern DisableNtfs <- ChatSettings {enableNtfs = False}
 
 data ChatFeature
   = CFTimedMessages
+  | CFReceiveReceipts
```

Comment:
CFDeliveryReceipts? doesn't matter

Reply:
the naming here reflects that it is the permission for another side - that is, the contact allows you to receive receipts. We can talk about it.

Reply:
Possibly it is more confusing than it could be...